chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/ChatManagerSpek.kt

```diff
@@ -177,14 +194,14 @@ class ChatManagerSpek : Spek({
 
             val room = pusherino.assumeSuccess().generalRoom
 
-            var messageReceived by FutureValue<com.pusher.chatkit.messages.multipart.Message>()
+            var messageReceived by FutureValue<Message>()
 
             pusherino.assumeSuccess().subscribeToRoomMultipart(room, RoomListeners(
                     onMultipartMessage = { message -> messageReceived = message },
                     onErrorOccurred = { e -> error("error: $e") }
             ))
 
-            val messageResult = alice.assumeSuccess().sendMessage(room, "message text")
+            val messageResult = alice.assumeSuccess().sendSimpleMessage(room, "message text")
```

Comment:
Not related  to V7 upgrade, but as related to the upgrade was fixing the tests to hit the newest backend API (unless explicitly intended to test legacy v2), that part was also by coincidence spotted and moved to use non-deprecated call.